base/stream.jl

```diff
@@ -876,6 +894,14 @@ function stop_reading(stream::LibuvStream)
     end
 end
 
+function readbytes!(s::LibuvStream, b::AbstractArray{UInt8}, nb=length(b))
+    wait_readnb(s, nb)
```

Comment:
This line breaks MbedTLS and everything that depends on it. I think `wait_readnb` should probably have a wider signature, `Integer` instead of `Int`? https://github.com/JuliaLang/julia/blob/0a00109270c09e307e692639cacce342b1098bec/base/stream.jl#L358

(also in need of tests)